services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -367,4 +456,108 @@ private void addValidIndividualRequest(final IBulkSubmission bulkSubmission, fin
         bulkSubmission.addIndividualRequest(individualRequest);
     }
 
+
+    @Test
+    public void setErrorMessagesCacheTest() throws Exception {
+        IGenericDao genericDaoMock = mock(GenericDao.class);
+        IBulkCustomerDao bulkCustomerDaoMock = mock(BulkCustomerDao.class);
+        ITargetApplicationDao targetApplicationDaoMock = mock(TargetApplicationDao.class);
+        IndividualRequestsXmlParser individualRequestsXmlParserMock = mock(IndividualRequestsXmlParser.class);
+        IMessagingUtility messagingUtility = mock(MessagingUtility.class);
+        ISdtBulkReferenceGenerator sdtBulkReferenceGeneratorMock = mock(SdtBulkReferenceGenerator.class);
+        ICacheable errorMessagesCache = Mockito.mock(ICacheable.class);
+
+
+        BulkSubmissionService bulkSubmission = new BulkSubmissionService(
+            genericDaoMock,bulkCustomerDaoMock, targetApplicationDaoMock,individualRequestsXmlParserMock,
+            messagingUtility,sdtBulkReferenceGeneratorMock,errorMessagesCache);
```

Comment:
Should be able to use the bulkSubmissionService field that's initialised in the setUp() method here.  This test just creates an instance of the bulk submission service then replaces the error message cache on it, so we might as well use the one that's already been set up.

Reply:
done